### PR TITLE
noninteractive-tradefed: add more debug

### PR DESCRIPTION
**Pull request by mwasilew**
Signed-off-by: Milosz Wasilewski <milosz.wasilewski@linaro.org>

### CODE REVIEW

File: automated/android/noninteractive-tradefed/monitor_fastboot.sh [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #!/bin/sh -x
 while true; do
```
Comment (chase-qi) [inline automated/android/noninteractive-tradefed/monitor_fastboot.sh]: Not this patch related, but looks like the script simply run 'fastboot boot'  every 30 seconds. I think we should at least check if a fastboot device appeared.

File: automated/android/noninteractive-tradefed/monitor_adb.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -x
+while true; do
+date
```
Comment (chase-qi) [inline automated/android/noninteractive-tradefed/monitor_adb.sh]: Indentation?

File: automated/android/noninteractive-tradefed/monitor_fastboot.sh (hunk 1 above)
Comment (mwasilew) [reply automated/android/noninteractive-tradefed/monitor_fastboot.sh]: fastboot will wait indefinitely for the device to appear. The whole idea is to boot the board if it goes to fastboot mode.

File: automated/android/noninteractive-tradefed/monitor_adb.sh (hunk 2 above)
Comment (mwasilew) [reply automated/android/noninteractive-tradefed/monitor_adb.sh]: will fix, thanks.

**Pull request merged**